src/pug/includes/body.pug

```diff
@@ -1,7 +1,9 @@
 include ./fork.pug
 
 .container
-
+	img(src="/assets/img/moon.png")#mode
+	.tooltip Enable dark mode
```

Comment:
I don't think we need the tooltip, it adds lot of logic for something that could simply be written on the icon side. WDYT?

Reply:
Yeah, I just thought it would be cool if it was more minimal. Should I get rid of it and just add a small text on the side?

Reply:
Is there a reason we can't use a toggle?

![image](https://user-images.githubusercontent.com/4383564/50704448-08d84d00-101d-11e9-8634-928178a7baf6.png)

Reply:
Toggle it is then!

Reply:
@vikaspotluri123 is there any native switch element?
I think we'd better go with a simple click to switch, like : 'make it dark', I don't see the need for an icon or something

Reply:
As mentioned in the issue, it's not overly complicated to implement, as it uses css to style a native checkbox input

We can definitely use text instead of icons 👍